ChatQnA/docker_compose/intel/cpu/xeon/set_env_mariadb.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2025 MariaDB Foundation
```

Comment:
This also should be ok, but I'm just noting it for myself to be confirmed with other stakeholders.